nifi-mock/src/main/java/org/apache/nifi/util/TestRunners.java

```diff
@@ -17,11 +17,12 @@
 package org.apache.nifi.util;
 
 import org.apache.nifi.processor.Processor;
+import org.apache.nifi.registry.VariableRegistryUtils;
 
 public class TestRunners {
 
     public static TestRunner newTestRunner(final Processor processor) {
```

Comment:
There should be the ability to customize the properties in the VariableRegistry. Maybe just give another static constructor that allows the user to pass it in and then forward that to the StandardProcessorTestRunner?